app/models/user.rb

```diff
@@ -1,11 +1,11 @@
 class User < ApplicationRecord
 
-  include ActionView::Helpers
```

Comment:
this overwrote the `image_url` method that would otherwise be needed. I don't see where these helpers are used, and the tests are passing, but perhaps it's good to do some manual testing too.